research-hub-web/cypress/integration/fundings.e2e.js

```diff
@@ -1,35 +1,31 @@
 describe('ResearchHubs Funding Pages', () => {
 
-  beforeEach(() => {
-      cy.visit('/funding/FRDF');
-  });
-
-  it('can visit a funding and display its title', () => {
-      cy.contains('Faculty Research Development Fund');
-  });
-
-  it('can visit a funding and display its subtitle', () => {
-      cy.contains('The Faculty Research Development Fund (FRDF)');
-  });
-
-  it('displays purpose text', () => {
-    cy.contains('The main objective of');
-});
-
-  it('displays body text', () => {
-      cy.contains('When thinking about the allocations of funds');
-  });
-
-  it('displays specifications table', () => {
-      cy.get('#specifications-table').contains('Details').should('exist');
-      cy.get('#specifications-table').contains('Description').should('exist');
-  });
-
-  it('displays a list of contacts', () => {
-      cy.get('#contacts').should('exist');
-  });
-
-  it('displays a list of organisations', () => {
-      cy.get('#organisations').should('exist');
-  });
+    beforeEach(() => {
+        cy.visit('/funding/FRDF');
+    });
+
+    it('can visit a case study and display its title', () => {
```

Comment:
Seems to be a few copy-paste errors in this file - should be funding, not case study

Reply:
It's weird that these tests are passing - it visits a funding page but this passes: cy.expect('app-case-study .content-summary').not.to.be.empty; 
The selector for the funding component is 'app-fundings' so I'm not sure if theses tests are robust.